crates/model/src/solver_competition.rs

```diff
@@ -7,11 +8,13 @@ use serde::{Deserialize, Serialize};
 use serde_with::serde_as;
 use std::collections::BTreeMap;
 
-pub type SolverCompetitionId = i64;
-
 #[derive(Clone, Debug, Default, Deserialize, Serialize, PartialEq)]
 #[serde(rename_all = "camelCase")]
 pub struct SolverCompetition {
+    // default annotation because until https://github.com/cowprotocol/services/pull/488 we weren't
+    // storing an auction id in the database.
+    #[serde(default)]
+    pub auction_id: AuctionId,
```

Comment:
This has the side effect of letting you see the auction id when querying solver competition by tx hash.